### ISSUE TITLE
复现

### ISSUE THREAD

**Issue by wsglm** [YYingDL/PCL-TDGCN#1]
你们的复现结果如何？

**Comment by IiCEight** [comment]
Hi, I only get 87% on SEED subject-independent. How about you?

**Comment by YYingDL** [comment]
Dear all,

I checked the code and revised line 340 as follows:

python
# Domain adversarial loss
global_transfer_loss = dann_loss(
    src_feature + 0.005 * torch.randn_like(src_feature).to(args.device),
    tar_feature + 0.005 * torch.randn_like(tar_feature).to(args.device),
    src_prob, tar_output_cls
)

**Comment by YYingDL** [comment]
We have found a bug and revised it.

**Comment by passionstar** [comment]
> We have found a bug and revised it.我们发现了这个漏洞，并已进行了

> Hi, I only get 87% on SEED subject-independent. How about you?嗨，我在 SEED 科目上的得分只有 87%。你呢？
现在您已经解决了吗？我也是只有这么多。但是我改了session 后，在session2的精度会高一点。

**Comment by IiCEight** [comment]
I solved this according to make the evaluation trigger at every epoch, current is every 10 epochs.

**Comment by passionstar** [comment]
> I solved this according to make the evaluation trigger at every epoch, current is every 10 epochs.我是通过在每个时代都触发评估来解决这个问题的，当前每 10 个时代进行一次评估。

好的，我也去调一下。谢谢你的回复。
